app/views/trainees/review_draft/show.html.erb

```diff
@@ -119,6 +119,17 @@
             ).status,
           )
 
+        if @trainee.requires_schools?
```

Comment:
Happy to pair with someone to bring all this logic from the view into the component 👍

Reply:
I think this is ok for now. It's consistent with the way we've done placement details. We can look at refactoring them together if we decide to do that.